Frontend/ui-driver/src/Flow.purs

```diff
@@ -1097,9 +1103,46 @@ homeScreenFlow = do
   let showGender = not (isJust (getGenderValue getDriverInfoResp.gender))
   let (Vehicle linkedVehicle) = (fromMaybe dummyVehicleObject getDriverInfoResp.linkedVehicle)
   setValueToLocalStore USER_NAME getDriverInfoResp.firstName
+
+  case getDriverInfoResp.mobileNumber of
```

Comment:
why do we have to do all of these operations whenever HomeScreenFlow is called?

Reply:
Fixed